actions/get_contents.py

```diff
@@ -7,8 +7,12 @@
 
 
 class GetContentsAction(BaseGithubAction):
-    def run(self, user, repo, ref, path, decode=False):
+    def run(self, user, repo, base_url, ref, path, decode=False):
 
+        if base_url == None:
+            self._reset(user)
+        else:
+            self._reset(user+'|'+base_url)
```

Comment:
nit: since we're repeating this code snippet in a number of places, it might be helpful to create a shared function to do this.

Reply:
Brandon, Thanks!  I'll refactor when I do the same for the pull request related actions.